packages/react-ui-ag/src/Listings/Listing.js

```diff
@@ -135,13 +140,7 @@ export default class Listing extends Component {
 
   renderCtaButton(props, key) {
     const { theme, listing } = this.props
-    const {
-      className,
-      onClick,
-      valueLocation,
-      children,
-      ...rest
-    } = props
+    const { className, onClick, valueLocation, children, ...rest } = props
```

Comment:
I personally thought these were easier to read on newlines, but to each their own i guess.

Reply:
I just have prettier format things on save. I can undo if you'd like. I don't have a preference.

Reply:
I think if you would of added one more item it would have broken it out onto their own lines lol

Reply:
I've seen prettier in some strange case put all but one prop on the same line, that one I really disagree with.

Reply:
Well, we have an eslint requirement in ag.js if it's 4 or more props, put them on new lines.  We will most likely be moving to prettier anyway, so I'm sure we can discuss it then.  @LinnJS I wonder if this has to do with max line length for breaking them into new lines?